### ISSUE TITLE
heketi volume creation should give "Mimimun volume size" instead of "Minimum brick size" error

### ISSUE THREAD

**Issue by NehaRawat** [heketi/heketi#409]
End user is providing volume size to the heketi-cli so it should provide "volume size" error instead of "brick size"

**Comment by obnoxxx** [comment]
@NehaRawat can you be more specific?

Reporting a minimum brick size error can be entirely valid if there are not enough bricks available that are big enough. So the request for a volume of requested size could only be satisfied by bricks that are smaller than allowed by MinimumBrickSize. Hence this would be working as designed.

**Comment by NehaRawat** [comment]
@obnoxxx during volume creation as a input we are providing volume size and not brick size. Heketi calculates the brick size based on "volume size" and returns MinimumBrickSize error. 

its working as designed but I think for end user "Minimum Volume Size" makes more sense as he is not aware of bricks sizes. Not a issue but can be enhancement.

**Comment by obnoxxx** [comment]
@NehaRawat But it does not need to be true that the requested volume size is too small: In the corner case the specified volume size itself is smaller than the minimum brick size (or currently smaller than 2* minimum brick size, or some other factor for disperse durability...), then it would be valid to report VolumeSize, but it can very well be that the requested volume size is big enough, only there are not enough big enough bricks left to build the volume. In this case some other error should be given.

If you think this level of detail in error reporting irritates the end user,  then we should probably return NoSpace and say that the cluster does not have enough space to accommodate the Volume. (Reason being in this case that the bricks needed to build the volume would be smaller than allowed).

Note: This behavior of returning NoSpace error instead of MinimumBrickSize and also instead of MaxBricks is introduced in PR #472 .

**Comment by obnoxxx** [comment]
Actually, after thinking about it more and looking at the code: heketi should report NoSpace already instead of MinBrickSize, even if MinBrickSize is used internally.

**Comment by lpabon** [comment]
@obnoxxx @NehaRawat is this still an issue or should will this be fixed by a PR?

**Comment by obnoxxx** [comment]
Oh, wasn't that closed by #506 ?

**Comment by obnoxxx** [comment]
```
[vagrant@storage0 ~]$ heketi-cli volume create --size=1
Error: Requested volume size (1 GB) is smaller than the minimum supported volume size (4194304)
```

So this is a duplicate of #506 
(or rather the other way around)
